src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -0,0 +1,95 @@
+            <th>{{ 'locationview.details.translations'|trans({}, 'locationview')|desc('Translations') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                {% if creator is not empty  %} {{ ez_content_name(creator) }} {% endif %}
+                ({{ contentInfo.publishedDate|localizeddate('medium', 'short', app.request.locale) }})
+            </td>
+            <td>
+                {% if lastContributor is not empty  %}{{ ez_content_name(lastContributor) }} {% endif %}
+                ({{ versionInfo.modificationDate|localizeddate('medium', 'short', app.request.locale) }})
+            </td>
+            <td>{{ versionInfo.versionNo }}</td>
+            <td>{{ translations|length }}
+                {% for translation in translations %}
+                    {{ translation.name }}<br>
+                {% endfor %}
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.technicaldetails'|trans({}, 'locationview')|desc('Technical details') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.contentid'|trans({}, 'locationview')|desc('Content ID') }}</th>
+            <th>{{ 'locationview.details.locationid'|trans({}, 'locationview')|desc('Location ID') }}</th>
+            <th>{{ 'locationview.details.contentremoteid'|trans({}, 'locationview')|desc('Content remote ID') }}</th>
+            <th>{{ 'locationview.details.remoteid'|trans({}, 'locationview')|desc('Location remote ID') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ contentInfo.id }}</td>
+            <td>{{ location.id  }}</td>
+            <td>{{ contentInfo.remoteId }}</td>
+            <td>{{ location.remoteId }}</td>
+        </tr>
+    </tbody>
+</table>
+
+
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.sectiondetails'|trans({}, 'locationview')|desc('Section details') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.sectionName'|trans({}, 'locationview')|desc('Section name') }}</th>
+            <th>{{ 'locationview.details.sectionIdentifier'|trans({}, 'locationview')|desc('Section identifier') }}</th>
+            <th>{{ 'locationview.details.sectionId'|trans({}, 'locationview')|desc('Section ID') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ section.name }}</td>
+            <td>{{ section.identifier }}</td>
+            <td>{{ section.id }}</td>
+        </tr>
+    </tbody>
+</table>
+
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.subitems.default.ordering'|trans({}, 'locationview')|desc('Sub-items default ordering') }}</h2>
```

Comment:
It would be better to set the translation domain globally for the template (`{% trans_default_domain "locationview" %}`, and omit the parameter in all `|trans` calls.